tests/test_driver.py

```diff
@@ -109,6 +111,45 @@ def test_send(self, driver, alice_privkey, unsigned_transaction):
         sent_tx = driver.transactions.send(fulfilled_tx)
         assert sent_tx == fulfilled_tx
 
+    def test_get_raises_type_error(self, driver):
+        """This test is somewhat important as it ensures that the
+        signature of the method requires the ``asset_id`` argument.
+        The http api would return a 400 if no ``asset_id`` is provided.
+
+        """
+        with raises(TypeError) as exc:
+            driver.transactions.get()
+        assert exc.value.args == (
+            "get() missing 1 required keyword-only argument: 'asset_id'",)
+
+    @mark.parametrize('query_params', (
+        {}, {'operation': 'CREATE'}, {'operation': 'TRANSFER'}
+    ))
+    def test_get_empty(self, driver, query_params):
+        response = driver.transactions.get(
+            asset_id=sha3_256(''.encode()).hexdigest())
```

Comment:
Well, ok, this is so to fit the requirement set by https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/web/views/parameters.py#L4 / https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/web/views/transactions.py#L59